numbat/src/resolver.rs

```diff
@@ -105,7 +110,7 @@ impl Resolver {
                                 &code,
                             );
 
-                            let imported_program = self.parse(&code, code_source_id)?;
+                            let imported_program = self.parse(&code, code_source_id, true)?;
```

Comment:
What is the reasoning for using `true` here, but not below?

Reply:
Hummm, when I wrote it initially, I thought that in most cases, we probably don't need it since we would be importing the prelude or stuff like that and thus want to « be faster ».

But actually, in the case it works, accumulating the errors costs nothing, I think; thus, maybe we should _always_ set it to `false`?

Should it even be configurable, actually?

Reply:
> Should it even be configurable, actually?

Ideally, no. Let's remove the argument, if possible :+1:

Reply:
Got rid of it; it simplified the code a little bit.